src/inc-lib-init.php

```diff
@@ -1084,336 +1084,488 @@ function lovd_getInstallURL ($bFull = true)
         if (!$aTranscriptOffsets[$sTranscriptID]) {
-            // Transcript not configured correctly.
-            // Don't die here; we might not even need these positions. We'll die later if we do.
+            // The transcript is not configured correctly. We will treat this transcript as unknown.
             $sTranscriptID = '';
         }
-    } elseif ($sTranscriptID === false) {
+
+    } elseif ($sTranscriptID == false) {
         // If the transcript ID is passed as false, we are asked to ignore not having the transcript.
         // Some random number, high enough to not be smaller than position_start if that's not in the UTR.
-        $aTranscriptOffsets[0] = 1000000;
-    }
-
-    // Isolate the position(s) from the variant. We don't support combined variants.
-    // We're not super picky, and would therefore approve of c.1_2A>C; we also
-    //  don't check for the end of the variant, it may contain bases, or not.
-    if (preg_match('/^([cgmn])\.(\()?([\-\*]?\d+)([-+](?:\d+|\?))?(?:_([\-\*]?\d+)([-+](?:\d+|\?))?)?([ACGT]>[ACGT]|con|del(?:ins)?|dup|inv|ins|\|(?:gom|lom|met=)|=)(.*)(?(2)\))/', $sVariant, $aRegs)) {
-        //             1 = Prefix; indicates what kind of positions we can expect, and what we'll output.
-        //                       2 = Do we have an opening parenthesis?
-        //                            3 = Start position, might be negative or in the 3' UTR.
-        //                                        4 = Start position intronic offset, if available.
-        //                                                             5 = End position, might be negative or in the 3' UTR.
-        //                                                                         6 = End position intronic offset, if available.
-        //                                                                                            7 = The variant, which we'll use to determine the type.
-        //                                                                                                                                 8 = The suffix.
-        list(, $sPrefix,, $sStartPosition, $sStartPositionIntron, $sEndPosition, $sEndPositionIntron, $sVariant, $sSuffix) = $aRegs;
-
-        if ($bCheckHGVS) {
-            // This was quite a lossy check, sufficient to get positions and type, but we need a HGVS check now.
-            if ($sVariant == 'con') {
-                // Conversion has been removed in favor of delins.
-                // See: http://varnomen.hgvs.org/bg-material/consultation/svd-wg009/
-                return false;
+        $aTranscriptOffsets[$sTranscriptID] = 1_000_000;
+    }
+    
+    
+    // Converting 3' UTR notations ('*' in the position fields) to normal notations,
+    //  and also checking for '?'.
+    foreach (array('earliest_start', 'latest_start', 'earliest_end', 'latest_end') as $sPosition) {
+        if (isset($aVariant[$sPosition][0]) && $aVariant[$sPosition][0] === '*') {
```

Comment:
Als ik het goed heb, bestaat `$aVariant[$sPosition]` altijd. In dat geval is `substr()` veel makkelijker in dit geval! (dan heb je de isset niet nodig)